table/table.go

```diff
@@ -51,41 +51,41 @@ const (
 
 var (
 	// ErrColumnCantNull is used for inserting null to a not null column.
-	ErrColumnCantNull  = terror.ClassTable.New(codeColumnCantNull, mysql.MySQLErrName[mysql.ErrBadNull])
-	errUnknownColumn   = terror.ClassTable.New(codeUnknownColumn, "unknown column")
-	errDuplicateColumn = terror.ClassTable.New(codeDuplicateColumn, "duplicate column")
+	ErrColumnCantNull  = terror.ClassTable.New(mysql.ErrBadNull, mysql.MySQLErrName[mysql.ErrBadNull])
+	errUnknownColumn   = terror.ClassTable.New(mysql.ErrBadField, mysql.MySQLErrName[mysql.ErrBadField])
+	errDuplicateColumn = terror.ClassTable.New(mysql.ErrFieldSpecifiedTwice, mysql.MySQLErrName[mysql.ErrFieldSpecifiedTwice])
```

Comment:
maybe need change the caller of `errUnknownColumn`, `errDuplicateColumn` due to "%" changed in message?

Reply:
But share same error code may be confused.